distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time_Formatter.enso

```diff
@@ -125,11 +125,12 @@ type Date_Time_Formatter
     @locale Locale.default_widget
     from_simple_pattern : Text -> Locale -> Date_Time_Formatter ! Date_Time_Format_Parse_Error
     from_simple_pattern pattern:Text locale:Locale=Locale.default =
-        FormatterCache.SIMPLE_FORMAT.get_or_set (FormatterCacheKey.new pattern locale.java_locale) _->
+        used_locale = if locale == Locale.default then Locale.us else locale
+        FormatterCache.SIMPLE_FORMAT.get_or_set (FormatterCacheKey.new pattern used_locale.java_locale) _->
```

Comment:
I think for consistency, we should add the same logic to `from_iso_week_date_pattern`. I'm not sure if it changes anything in there (maybe in the subset that uses the US and ROOT locales are the same, but I wouldn't want to risk any differences causing inconsistency).